### ISSUE TITLE
NTR: Sulfotransferase activity decreased in cultured fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8219]
New term request Sulfotransferase activity decreased in cultured fibroblasts (elthat_110711025518):

**Comment by pnrobinson** [comment]
@MickeySegal
there are many enzymes with sulfotransferase activity. What is the context?

**Comment by MickeySegal** [comment]
It is Carbohydrate sulfotransferase 3 as in https://www.ncbi.nlm.nih.gov/books/NBK62112/

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000970
- label: Diminished sulfotransferase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:18513679
- def: Rate of proteoglycan sulfation (sulfotransferase activity) is below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8219
- synonyms:
